### ISSUE TITLE
This is literally virtualenvwrapper

### ISSUE THREAD

**Issue by barseghyanartur** [burakyilmaz321/benv#11]
FYI, a lot of functionality overlap. 

https://virtualenvwrapper.readthedocs.io/en/latest/

**Comment by burakyilmaz321** [comment]
Yeah, I know. This is not an actual project. Just for learning and fun. Thanks!
